<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DetectorKZWidget</class>
 <widget class="QWidget" name="DetectorKZWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>532</width>
    <height>237</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DetectorKZWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,1,1,1">
     <property name="spacing">
      <number>20</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QPushButton" name="connectDetectorBtn">
         <property name="minimumSize">
          <size>
           <width>71</width>
           <height>35</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>71</width>
           <height>35</height>
          </size>
         </property>
         <property name="text">
          <string>连接探测器</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="setModeBtn">
         <property name="minimumSize">
          <size>
           <width>71</width>
           <height>35</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>71</width>
           <height>35</height>
          </size>
         </property>
         <property name="text">
          <string>设置参数</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="minimumSize">
            <size>
             <width>71</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>71</width>
             <height>25</height>
            </size>
           </property>
           <property name="text">
            <string>触发方式：</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="trigComboBox">
           <property name="minimumSize">
            <size>
             <width>61</width>
             <height>35</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>61</width>
             <height>35</height>
            </size>
           </property>
           <property name="currentIndex">
            <number>0</number>
           </property>
           <item>
            <property name="text">
             <string>内触发</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>外触发</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="minimumSize">
            <size>
             <width>71</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>71</width>
             <height>25</height>
            </size>
           </property>
           <property name="text">
            <string>增益等级：</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="gainComboBox">
           <property name="minimumSize">
            <size>
             <width>45</width>
             <height>35</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>45</width>
             <height>35</height>
            </size>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <item>
            <property name="text">
             <string>4</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>5</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>6</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>7</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label">
           <property name="minimumSize">
            <size>
             <width>71</width>
             <height>21</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>71</width>
             <height>21</height>
            </size>
           </property>
           <property name="text">
            <string>图像尺寸：</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="modeComboBox">
           <property name="minimumSize">
            <size>
             <width>85</width>
             <height>35</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>85</width>
             <height>35</height>
            </size>
           </property>
           <item>
            <property name="text">
             <string>704*704</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>1408*1408</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2816*2816</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>3</number>
         </property>
         <item>
          <widget class="QLabel" name="label_8">
           <property name="minimumSize">
            <size>
             <width>51</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>51</width>
             <height>25</height>
            </size>
           </property>
           <property name="text">
            <string>帧  率：</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="frameSpinBox">
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>35</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>35</height>
            </size>
           </property>
           <property name="maximum">
            <double>15.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.100000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_6">
           <property name="minimumSize">
            <size>
             <width>31</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>31</width>
             <height>25</height>
            </size>
           </property>
           <property name="text">
            <string> 帧/s</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="minimumSize">
            <size>
             <width>71</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>71</width>
             <height>25</height>
            </size>
           </property>
           <property name="text">
            <string>曝光时间：</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="exposureSpinBox">
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>35</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>35</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_7">
           <property name="minimumSize">
            <size>
             <width>21</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>21</width>
             <height>25</height>
            </size>
           </property>
           <property name="text">
            <string> ms</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_11">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QSpinBox" name="picNumSpinBox">
           <property name="minimumSize">
            <size>
             <width>51</width>
             <height>35</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>51</width>
             <height>35</height>
            </size>
           </property>
           <property name="maximum">
            <number>1000</number>
           </property>
           <property name="value">
            <number>50</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="minimumSize">
            <size>
             <width>15</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>15</width>
             <height>25</height>
            </size>
           </property>
           <property name="text">
            <string> 帧</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_14">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_12">
           <property name="minimumSize">
            <size>
             <width>40</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>40</width>
             <height>25</height>
            </size>
           </property>
           <property name="text">
            <string>转动：</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="rMoveSpinBox">
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>35</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>35</height>
            </size>
           </property>
           <property name="maximum">
            <double>360.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_13">
           <property name="minimumSize">
            <size>
             <width>25</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>25</width>
             <height>25</height>
            </size>
           </property>
           <property name="text">
            <string>rad</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_12">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_10">
           <property name="minimumSize">
            <size>
             <width>40</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>40</width>
             <height>25</height>
            </size>
           </property>
           <property name="text">
            <string>转速：</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="rSpeedSpinBox">
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>35</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>35</height>
            </size>
           </property>
           <property name="maximum">
            <double>1000.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_15">
           <property name="minimumSize">
            <size>
             <width>35</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>35</width>
             <height>25</height>
            </size>
           </property>
           <property name="text">
            <string> rad/s</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="darkAdjustBtn">
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>35</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>35</height>
          </size>
         </property>
         <property name="text">
          <string>暗场校准</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="gainAdjustBtn">
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>35</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>35</height>
          </size>
         </property>
         <property name="text">
          <string>增益校准</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <property name="spacing">
        <number>3</number>
       </property>
       <item>
        <widget class="QComboBox" name="getPicModeComboBox">
         <property name="minimumSize">
          <size>
           <width>71</width>
           <height>35</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>71</width>
           <height>35</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>空气曝光</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>几何校正</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>连续采集</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>步进采集</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="getPicBtn">
         <property name="minimumSize">
          <size>
           <width>71</width>
           <height>35</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>71</width>
           <height>35</height>
          </size>
         </property>
         <property name="text">
          <string>模式取图</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="startPulseBtn">
         <property name="minimumSize">
          <size>
           <width>71</width>
           <height>35</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>71</width>
           <height>35</height>
          </size>
         </property>
         <property name="text">
          <string>连续取图</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="stopPulseBtn">
         <property name="minimumSize">
          <size>
           <width>71</width>
           <height>35</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>71</width>
           <height>35</height>
          </size>
         </property>
         <property name="text">
          <string>停止取图</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_11">
         <property name="minimumSize">
          <size>
           <width>71</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>71</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string>已 采 图：</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="picNumLabel">
         <property name="minimumSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string>0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_9">
         <property name="minimumSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string> 帧</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
